float32[] joints_input
geometry_msgs/Pose target_pose
---
moveit_msgs/RobotTrajectory trajectory
